### PR TITLE
Updated Matrix.dot() Docstring

### PR DESCRIPTION
**Pull request by ddjr**
Updated Matrix.dot() Docstring by adding parameters as well as updating variable names with `` notation.

### CODE REVIEW

File: Untitled.ipynb [hunk 1]
```diff
@@ -0,0 +1,254 @@
+{
```
Comment (siefkenj) [inline Untitled.ipynb]: You shouldn't be committing this file.

File: sympy/matrices/matrices.py [hunk 2]
```diff
@@ -4319,13 +4323,17 @@ def diff(self, *args):
                          lambda i, j: self[i, j].diff(*args))
 
     def dot(self, b):
-        """Return the dot product of Matrix self and b relaxing the condition
+        """Return the dot product of Matrix ``self`` and ``b`` relaxing the condition
         of compatible dimensions: if either the number of rows or columns are
-        the same as the length of b then the dot product is returned. If self
+        the same as the length of ``b`` then the dot product is returned. If self
```
Comment (siefkenj) [inline sympy/matrices/matrices.py]: If you're double quoting `self`, double quote all instances.

**Review by siefkenj — CHANGES_REQUESTED** [review]

**Comment by gxyd** [comment]
@ddjr can you please address the requested changes?

**Comment by gxyd** [comment]
This PR has been inactive for quite sometime, so we are going to close it and open an issue to reference it. We try to keep our pull request queue small and focused on active work.  We encourage you to reopen the pull request if and when you continue to work on this. Please contact us if you have any questions.

Thanks for contributing.

see https://github.com/sympy/sympy/wiki/Closing-pull-requests for our policies on closing pull request.

I've opened https://github.com/sympy/sympy/issues/13449 for this pull request so that we don't forget about it.

**Pull request closed without merging**